core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -214,7 +205,7 @@ object Boot extends IOApp with LazyLogging {
       }
 
       val executionServiceConfig = conf.getConfig("executionservice")
-      val submissionTimeout = util.toScalaDuration(
+      val submissionTimeout = org.broadinstitute.dsde.rawls.util.toScalaDuration(
```

Comment:
not technically part of this PR; IntelliJ showed an error here until the full package was spelled out.